apitesters/paywalls.tsx

```diff
@@ -1,25 +1,66 @@
-import RNPaywalls from "../react-native-purchases-ui/";
-import { Text } from "react-native";
 import React from 'react';
 
+import RevenueCatUI, {
+  PAYWALL_RESULT,
+  PresentPaywallIfNeededParams,
+  PresentPaywallParams
+} from "../react-native-purchases-ui";
+
 async function checkPresentPaywall() {
-  await RNPaywalls.presentPaywall();
+  let paywallResult: PAYWALL_RESULT = await RevenueCatUI.presentPaywall({});
+  paywallResult = await RevenueCatUI.presentPaywall();
+  paywallResult = await RevenueCatUI.presentPaywall({
+    // offeringIdentifier: "offering",
+  });
+  paywallResult = await RevenueCatUI.presentPaywall({
+    displayCloseButton: false,
+  });
+  paywallResult = await RevenueCatUI.presentPaywall({
+    // offeringIdentifier: "offering",
+    displayCloseButton: false,
+  });
 }
 
 async function checkPresentPaywallIfNeeded() {
-  await RNPaywalls.presentPaywallIfNeeded("entitlement");
+  let paywallResult: PAYWALL_RESULT = await RevenueCatUI.presentPaywallIfNeeded({
+    requiredEntitlementIdentifier: "entitlement"
+  });
+  paywallResult = await RevenueCatUI.presentPaywallIfNeeded({
+    requiredEntitlementIdentifier: "entitlement",
+    // offeringIdentifier: "offering",
+  });
+  paywallResult = await RevenueCatUI.presentPaywallIfNeeded({
+    requiredEntitlementIdentifier: "entitlement",
+    displayCloseButton: false,
+  });
+  paywallResult = await RevenueCatUI.presentPaywallIfNeeded({
+    requiredEntitlementIdentifier: "entitlement",
+    // offeringIdentifier: "offering",
+    displayCloseButton: false,
+  });
+}
+
+function checkPresentPaywallParams(params: PresentPaywallIfNeededParams) {
+  const requiredEntitlementIdentifier: string = params.requiredEntitlementIdentifier;
+  // const offeringIdentifier: string | undefined = params.offeringIdentifier;
+  const displayCloseButton: boolean | undefined = params.displayCloseButton;
+}
+
+function checkPresentPaywallIfNeededParams(params: PresentPaywallParams) {
+  // const offeringIdentifier: string | undefined = params.offeringIdentifier;
+  const displayCloseButton: boolean | undefined = params.displayCloseButton;
 }
 
 const PaywallScreen = () => {
   return (
-    <RNPaywalls.Paywall style={{ marginBottom: 10 }}/>
+    <RevenueCatUI.Paywall style={{marginBottom: 10}}/>
```

Comment:
Interesting, was this not failing before?

Reply:
It was the same, I changed the import from `RNPaywalls` to `RevenueCatUI` in line 3, you can name it anything

Reply:
Oooh hahha right right